setup.py

```diff
@@ -53,6 +53,8 @@ def version(path):
     "nbformat",
     "nbconvert<7.0.0",
     "requests",
+    "fsspec",
+    "universal_pathlib",
```

Comment:
https://github.com/fsspec/universal_pathlib seems to have small number of stars? Is this a well maintained lib?

Also if we _have_ to use it and it's not a very popular lib, we should pin its version to avoid complications.

Reply:
Found a bug that universal_pathlib is messing up with storage protocol. I've removed it from dependency now.